Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -411,7 +411,15 @@ AccessibilityRole AccessibilityNodeObject::determineAccessibilityRole()
     if (isExposedTableRow())
         return AccessibilityRole::Row;
 
-    return determineAccessibilityRoleFromNode();
+    AccessibilityRole roleFromNode = determineAccessibilityRoleFromNode();
```

Comment:
`auto` seems appropriate here considering the variable name and method initializing the variable have "role" in it?